### PR TITLE
Remove partition before destroying efi (bnc#929677)

### PR DESCRIPTION
**Pull request by ancorgs**
This is a port from the fix to bnc#870211 (SLE-11-SP4). Just copy&paste and some minor fixes.

This depends on the version of efibootmgr having the required feature (already commented in the bug report).

### CODE REVIEW

**Comment by jreidinger** [comment]
looks good from my perspective

File: package/libstorage.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Jul 10 10:07:26 UTC 2015 - ancor@suse.com
+
+- remove partition from efi before destroying it on GPT disks
+  (bnc#929677)
```
Comment (aschnell) [inline package/libstorage.changes]: Lets hope the bug tracking scripts can handle the bug split and know that #870211 is also fixed.

**Comment by aschnell** [comment]
LGTM

File: package/libstorage.changes (hunk 1 above)
Comment (jreidinger) [inline package/libstorage.changes]: good note, I suggest to place both bug numbers

**Pull request merged**